<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FakeVimOptionPage</class>
 <widget class="QWidget" name="FakeVimOptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>322</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="checkBoxUseFakeVim">
     <property name="text">
      <string>Use FakeVim</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Vim style settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="labelExpandTab">
        <property name="toolTip">
         <string>vim's &quot;expandtab&quot; option</string>
        </property>
        <property name="text">
         <string>Expand tabulators:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBoxExpandTab">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelHlSearch">
        <property name="text">
         <string>Highlight search results:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBoxHlSearch">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelShiftWidth">
        <property name="text">
         <string>Shift width:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="lineEditShiftWidth"/>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="labelSmartTab">
        <property name="text">
         <string>Smart tabulators:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QCheckBox" name="checkBoxSmartTab">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelStartOfLine">
        <property name="text">
         <string>Start of line:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QCheckBox" name="checkBoxStartOfLine">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_2">
        <property name="toolTip">
         <string>vim's &quot;tabstop&quot; option</string>
        </property>
        <property name="text">
         <string>Tabulator size:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="lineEditTabStop"/>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="labelBackspace">
        <property name="text">
         <string>Backspace:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="lineEditBackspace"/>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBoxAutoIndent">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelAutoIndent">
        <property name="toolTip">
         <string>VIM's &quot;autoindent&quot; option</string>
        </property>
        <property name="text">
         <string>Automatic indentation:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelIncSearch">
        <property name="text">
         <string>Incremental search:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBoxIncSearch">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonCopyTextEditorSettings">
       <property name="text">
        <string>Copy text editor settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSetQtStyle">
       <property name="text">
        <string>Set Qt style</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSetPlainStyle">
       <property name="text">
        <string>Set plain style</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
